dataprofiler/profilers/categorical_column_profile.py

```diff
@@ -5,6 +5,7 @@
 from operator import itemgetter
 from typing import cast
 
+from datasketches import count_min_sketch
```

Comment:
instead, let's do 
```python
import datasketches
```

elsewhere we can do:
```python
datasketches.count_min_sketch
```

Otherwise, we don't know the lineage of `count_min_sketch` for clairity as it is external.

Reply:
gotcha!